part-ii-domain-logic/deadlines.md

```diff
@@ -0,0 +1,63 @@
+# Deadlines
+
+This feature is introduced in 3.3 version of Axon. Until deadlines, it was possible to schedule an event in a Saga. If you wish more control when a deadline is met, this is the feature to be used. 
```

Comment:
IMO it is interesting to know since when this feature exists. Otherwise, you'd have to click through the versions and notice when it is introduced. What do you think?

Reply:
After all, I think that you're right about the version mentioning. I'll remove it.